docs/getting-started/running-release.md

```diff
@@ -116,14 +115,15 @@ kind create cluster --name=test-bpfman
 Next, deploy the bpfman CRDs:
 
 ```console
-export BPFMAN_REL=0.3.1
-kubectl apply -f  https://github.com/bpfman/bpfman/releases/download/v${BPFMAN_REL}/bpfd-crds-install-v${BPFMAN_REL}.yaml
+export BPFMAN_REL=0.4.0
+kubectl apply -f  https://github.com/bpfman/bpfman/releases/download/v${BPFMAN_REL}/bpfman-crds-install-v${BPFMAN_REL}.yaml
```

Comment:
I should probably fix this up in the release payload no need to have 

```
releases/download/v${BPFMAN_REL}/bpfman-crds-install-v${BPFMAN_REL}.yaml
```
versus just 

```
releases/download/v${BPFMAN_REL}/bpfman-crds-install.yaml
```

Reply:
Changed here. #1056 to track change in the release payload.